adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -278,38 +262,24 @@ - (BOOL)hasDelegateForPlacementID:(nonnull NSString *)placementID {
   return NO;
 }
 
-- (BOOL)canRequestBannerAdForPlacementID:(nonnull GADMAdapterVungleBannerRequest *)bannerRequest {
-  @synchronized(_bannerDelegates) {
-    if (_bannerDelegates.count > 0) {
-      [self checkSamePlacementIDwithNilUniquePubRequestID:bannerRequest];
-      return _bannerRequest == nil || [_bannerRequest isEqualToBannerRequest:bannerRequest];
-    }
-    return YES;
-  }
-}
-
 - (nullable NSError *)loadAd:(nonnull NSString *)placement
                 withDelegate:(nonnull id<GADMAdapterVungleDelegate>)delegate {
+  if (!delegate) {
+    return GADMAdapterVungleErrorWithCodeAndDescription(
```

Comment:
Can remove. This input is nonnull input and only adapter classes call this method. This code only masks an adapter bug that's not respecting nonnull, whic h

If you found a reason why this needs to be here, please fix the caller to not pass in a nil delegate.

Reply:
@ericleich Got it, I will remove this non nil check. Thanks a lot for the instruction. ^_^

Reply:
@ericleich Removed, thanks and have a great day! 😊